.pylintrc

```diff
@@ -36,6 +86,19 @@ load-plugins=
 # Pickle collected data for later comparisons.
 persistent=yes
 
+# Minimum Python version to use for version dependent checks. Will default to
+# the version used to run pylint.
+py-version=3.11
```

Comment:
Wouldn't this break compatibility with 3.6?

Reply:
fixed in 38d079d0a8aeb